src/navigation/navigationhandler.cpp

```diff
@@ -100,6 +100,16 @@ namespace {
         "again.",
         openspace::properties::Property::Visibility::User
     };
+
+    constexpr openspace::properties::Property::PropertyInfo FadeJumpThresholdInfo = {
+        "FadeJumpThreshold",
+        "Fade Jump Threshold",
+        "The threshold that determines when a transition should fade instead of "
```

Comment:
Clarify that this goes for a transition when setting the _time_

However, I still feel like we should remove this property and feature. If we keep it, I have a feeling that it should be applied in more cases where we set the time, but that would be too confusing.